requirements.in

```diff
@@ -2,10 +2,9 @@ docker
 PyYAML
```

Comment:
We should probably remove `requirements.in` and `requirements.txt` altogether.

Reply:
dropped